MARS_Specflow/ProfilePage.cs

```diff
@@ -0,0 +1,258 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.BiDi.Modules.Log;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MARS
+{
+    public class ProfilePage
+    {
+        IWebDriver driver;
+
+        public ProfilePage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public void AddingNewLanguage(string language, string level)
```

Comment:
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html